src/api/app/views/webui2/webui/package/revisions.html.haml

```diff
@@ -5,8 +5,8 @@
   = render(partial: 'tabs', locals: { project: @project, package: @package })
 
   .card-body
-    - if @revisions
-      %h3 #{@pagetitle} (#{@revisions.count})
+    %h3 #{@pagetitle} (#{@revisions.count})
+    - if @revisions.present?
```

Comment:
`present` would mean that this could also be nil. In that case the previous count would cause an error. From the commit message it seems like @revisions would be an array. So how about `any?`

Reply:
On second thought... couldn't we just always show the revisions? Unless the empty `.list-group` messes up the view, I would go with that.

Reply:
In comparison to `present?`, `any?` doesn't make a difference when checking empty arrays. However, when checking arrays with elements (especially huge arrays), then it can be slower since it goes through the elements until finding one which satisfies the condition. In this specific case with revisions, it doesn't apply. Still, I believe it's better to use `present?` in all cases because when it matters, then you don't have to wonder if you should use it or not.

Reply:
It's a bit weird if we don't display anything beside the title on the page when there are no revisions. I prefer to keep the message

Reply:
Ah, right. `any?` doesn't make sense here. I still find the `present?` misleading, but it's just a nitpick